README.md

```diff
@@ -114,7 +219,28 @@ Imagine that the squares of an ordinary checkerboard are numbered in two differe
 Write an expression to set a variable _color_ to either 'black' or 'white', describing the square identified by variables R,C, and N.
 (_Hint: If you prefer, you may use a pair of conditional statements instead of a single expression._)
 
----
+<pre>
+    var width = 8;     //width (and height) of checkerboard
```

Comment:
Nice job including a variable for `width`; that makes your code more flexible than having the value 8 hardcoded into it.